### PR TITLE
Create codeql.yml

### PR DESCRIPTION
**Pull request by Behnam-Shobiri**
Create CodeQL that would run on every pull_request for the "master" and "release-v1.1" branches.
It would also run every week on Sundays.

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master", "release-v1.1" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master", "release-v1.1" ]
+  schedule:
+    - cron: '0 23 * * 0'
```
Comment (rene-dekker) [inline .github/workflows/codeql.yml:21]: nit: If this is UTC, we may want to change 23 -> 3

**Review by rene-dekker — APPROVED** [review]
lgtm

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**